TAF/utils/scripts/docker/docker-compose-end-to-end.yaml

```diff
@@ -44,7 +44,8 @@
       Database_Username: appservice
       Database_Password: password
       EDGEX_SECURITY_SECRET_STORE: "false"
-      Writable_Pipeline_Functions_MQTTSend_Addressable_Address: mqtt-broker
+      Writable.Pipeline.Functions.MQTTSecretSend.Parameters.brokeraddress: tcp://mqtt-broker:1883
+      Writable_Pipeline_Functions_MQTTSecretSend_Parameters_topic: edgex-events
```

Comment:
is the topic called mytopic?
https://github.com/edgexfoundry/app-service-configurable/pull/145/files#diff-4458c498bf808b3986341cd108b57fdd08a6ca15b9f83f88cea82d2e54d0644cR60

Reply:
The topic set to mytopic on default toml file, but [mqtt subscriber](https://github.com/edgexfoundry/edgex-taf/blob/3fb5004859290821867256b0ebefbda7a4e81341/TAF/utils/src/setup/mqtt-subscriber.py#L12) listens topic "edgex-events".